kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ProxyDeployment.java

```diff
@@ -107,7 +107,7 @@ private static Container proxyContainer(KafkaProxy primary,
         // @formatter:off
         var containerBuilder = new ContainerBuilder()
                 .withName("proxy")
-                .withImage("quay.io/kroxylicious/kroxylicious:0.9.0-SNAPSHOT")
+                .withImage("quay.io/kroxylicious/kroxylicious:0.10.0-SNAPSHOT")
```

Comment:
I was thinking it would be nice if the proxy version came in via configuration, but it's not a quick fix. Because of the way we use a managed workflow where the JOSDK instantiates the secondary resource reconcilers for us we don't have somewhere to inject configuration. Currently we pass the RuntimeDecl it through the managed context, but maybe we should be looking at switching from managed workflow to building it up programatically so we can construct ProxyDeployment with a configuration object.

If we had it come in via config we could have a default config yaml on the classpath and make the release script update it. That would make the release scripting one command to update all yaml under `kroxylicious-operators`.